lib/response.js

```diff
@@ -718,6 +718,7 @@ res.header = function header(field, val) {
 
     // add charset to content-type
     if (field.toLowerCase() === 'content-type' && !charsetRegExp.test(value)) {
+      if (typeof value !== 'string')  throw "Please provide a string for content-type.";
```

Comment:
Please throw error objects, not primitive values. Ideally the error message should be in a similar form to others in the code base (i.e. no "Please", no trailing".", etc.).